client/src/components/Form/Elements/FormSelect.vue

```diff
@@ -0,0 +1,182 @@
+<script setup lang="ts">
+import { computed, onMounted, watch, type ComputedRef } from "vue";
+import Multiselect from "vue-multiselect";
+
+type SelectValue = string | null;
+
+interface SelectOption {
+    label: string;
+    value: SelectValue;
+}
+
+const props = withDefaults(
+    defineProps<{
+        multiple?: boolean;
+        optional?: boolean;
+        options: Array<[string, string]>;
```

Comment:
Bit of a theoretical concern if most select options are defined in XML, but it would be very nice if we could use other data types as well.